tests/modules/annotations/resources/utils.py

```diff
@@ -11,13 +11,20 @@
 
 
 def create_annotation_simple(
-    flask_app_client, user, asset_uuid, expected_status_code=200
+    flask_app_client,
+    user,
+    asset_uuid,
+    ia_class='test',
```

Comment:
Rather than two methods why don't you have one create_annotation(flask_app_client, user, data, expected_status_code=200) and have he calling code know what it needs to populate

Reply:
there were a few tests which i didnt originally write, which relied on the creation of an encounter before the annotation could be made (as this _used to be_ a requirement).  i just was side-stepping changing these basically.   i actually started switching over a couple of them and some tests failed so i backed out.  (tho, indeed, those failures may have been unrelated -- i never went back and revisited)